.githooks/pre-commit

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+STASH_NAME="pre-commit-$(date +%s)"
+git stash save -q --keep-index $STASH_NAME
+
+CHANGED_DTHS=$(git diff --cached --name-only | grep .*.groovy)
```

Comment:
It looks like you didn't end up using this elsewhere other than the commented out line below.